src/main/java/com/google/devtools/build/lib/syntax/Lexer.java

```diff
@@ -457,10 +469,14 @@ private void escapedStringLiteral(char quot, boolean isRaw) {
             case 'v':
             case 'x':
               // exists in Python but not implemented in Blaze => error
-              error("escape sequence not implemented: \\" + c, literalStartPos, pos);
+              error("invalid escape sequence: \\" + c, literalStartPos, pos);
               break;
             default:
               // unknown char escape => "\literal"
+              stringEscapeEvents.add(Event.error(
+                      createLocation(pos - 1, pos), "invalid escape sequence: \\" + c)
```

Comment:
Error message should mention `incompatible_restrict_string_escapes`.

Reply:
Done